src/vs/workbench/contrib/terminalContrib/find/browser/terminal.find.contribution.ts

```diff
@@ -174,23 +196,24 @@ registerActiveInstanceAction({
 		}
 	],
 	precondition: ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated),
-	run: (activeInstance) => {
-		const widget = TerminalFindContribution.get(activeInstance)?.findWidget;
+	run: (_xterm, _accessor, activeInstance) => {
+		const contr = TerminalFindContribution.activeFindWidget || TerminalFindContribution.get(activeInstance);
+		const widget = contr?.findWidget;
 		if (widget) {
 			widget.show();
 			widget.find(false);
 		}
 	}
 });
 
-registerActiveInstanceAction({
+registerActiveXtermAction({
```

Comment:
Was this rename an accident?

Reply:
`registerActiveXtermAction` is a variation that works for both detached and attached terminals. `registerActiveInstanceAction` only applies to the attached `ITerminalInstance`